packages/shared/types.ts

```diff
@@ -538,6 +538,13 @@ export interface GetFriendsPayload {
   skip: number // amount of entries to skip
 }
 
+export interface GetFriendRequestsPayload {
+  sentLimit: number // max amount of entries to receive
+  sentSkip: number // the amount of sent requests to skip
+  receivedLimit: number // max amount of entries to receive
```

Comment:
What's the difference with the `sentLimit`, let's clarify that for the four fields.

Reply:
received are the requests that you have received, only the description is wrong